ironfish/src/rpc/routes/wallet/postTransaction.test.ts

```diff
@@ -27,6 +27,7 @@ describe('Route wallet/postTransaction', () => {
 
     expect(addSpy).toHaveBeenCalledTimes(0)
     expect(response.status).toBe(200)
+    expect(response.content.hash).toBeDefined()
```

Comment:
Is it worth it to calculate what the `hash` should be and check equality, do you think?

Reply:
It's a little tricky to do this: there's randomness involved in posting transactions, so posting the same raw transaction twice will result in two different hashes 😓

Reply:
Ah, true. Forget it, then, imo.